### PR TITLE
Dqm single electron jet

### PR DESCRIPTION
**Pull request by parbol**
Changes as suggested in the email. Everything I did moved to SUSY area.

### CODE REVIEW

File: DQMOffline/Trigger/python/TopMonitor_cfi.py [hunk 1]
```diff
@@ -59,7 +59,7 @@
 #pt binning
 hltTOPmonitoring.histoPSet.elePtBinning = cms.vdouble(0,5,10,20,30,40,50,70,100,200,400)
 hltTOPmonitoring.histoPSet.jetPtBinning = cms.vdouble(0,5,10,20,30,40,50,70,100,200,400)
-hltTOPmonitoring.histoPSet.muPtBinning  = cms.vdouble(0,5,10,20,30,40,50,70,100,200,400)
+ltTOPmonitoring.histoPSet.muPtBinning  = cms.vdouble(0,5,10,20,30,40,50,70,100,200,400)
```
Comment (defranchis) [inline DQMOffline/Trigger/python/TopMonitor_cfi.py]: @parbol mind that you introduced a typo here. maybe you can check-out the file from the branch to make sure everything is the same as before for Top modules

File: DQMOffline/Trigger/python/SusyMonitor_cfi.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+hltSUSYmonitoring.numGenericTriggerEventPSet.andOrHlt      = cms.bool(True)# True:=OR; False:=AND
+hltSUSYmonitoring.numGenericTriggerEventPSet.hltInputTag   = cms.InputTag( "TriggerResults::HLT" )
+hltSUSYmonitoring.numGenericTriggerEventPSet.errorReplyHlt = cms.bool( False )
+hltSUSYmonitoring.numGenericTriggerEventPSet.verbosityLevel = cms.uint32(0)
+
+hltSUSYmonitoring.denGenericTriggerEventPSet.andOr         = cms.bool( False )
+hltSUSYmonitoring.numGenericTriggerEventPSet.andOrHlt      = cms.bool(True)# True:=OR; False:=AND
+hltSUSYmonitoring.numGenericTriggerEventPSet.hltInputTag   = cms.InputTag( "TriggerResults::HLT" )
+hltSUSYmonitoring.numGenericTriggerEventPSet.errorReplyHlt = cms.bool( False )
+hltSUSYmonitoring.denGenericTriggerEventPSet.dcsInputTag   = cms.InputTag( "scalersRawToDigi" )
```
Comment (defranchis) [inline DQMOffline/Trigger/python/SusyMonitor_cfi.py:100]: @parbol lines 97-99 should be "denGenericTriggerEventPSet". I know the bug comes from my code (which should be fixed now), sorry about this

**Comment by defranchis** [comment]
@parbol in addition to the comments in the code, please note that if you are interested in efficiency plots you should also add your code in https://github.com/cms-sw/cmssw/blob/master/DQMOffline/Trigger/python/SusyMonitoring_Client_cff.py in a similar way to https://github.com/defranchis/cmssw/blob/BTV-DQMOffline/DQMOffline/Trigger/python/TopMonitoring_Client_cff.py . Otherwise you only create final state distributions

**Comment by defranchis** [comment]
@parbol and please checkout from my branch DQMOffline/Trigger/python/TopMonitor_cfi.py and DQMOffline/Trigger/python/TopMonitoring_cff.py to restore to the original version

**Comment by parbol** [comment]
Hi Matteo ,

Thank you very much for your help. It's really appreciated. I think I have corrected all of them. I just have one remaining question. For the module doing the efficiency: do I need to get one module for every ele_jet trigger I want to monitor? Or it's just enough to indicate the folder and it will take them from there.

Thanks!

**Pull request merged**